### PR TITLE
setup

### PR DESCRIPTION
**Pull request by summyalena**
### I created and set up the react project.
- I ensured i setup the react project without issues pending
- I followed github flow.
- I ensured there were no linter errors.

### CODE REVIEW

**Comment by rloterh** [comment]
@summyalena 
Kindly ensure to remove the Logo from your ReadMe!

File: src/components/greetings/greeting.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    dispatch(fetchData);
+    console.log('hello');
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h3>Hello everyone</h3>
+      {greetings.message}
+    </div>
+  );
```
Comment (OLIPLICHE) [inline src/components/greetings/greeting.js]: - Hello, Here is what I am getting after connecting rails Api and React. I would advise you to display random greetings and submit them again for review.
-  NB: The servers are running perfectly
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @summyalena, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Alena,  

Your project is complete!🏁 Great job for nailing it! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- React frontend connects correctly to Rails API backend ❣️
- Random greetings is generating correctly ❣️ 

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Please bear in mind that the API backend project should be used for submission of this project. 
- Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), or optionally remove this section.
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**